main.go

```diff
@@ -102,57 +109,66 @@ func LoginHandler(w http.ResponseWriter, r *http.Request) {
 	email := d.Email
 	password := d.Password
 
-
 	if email == "" {
-		error.Message = "Email は,必須です。"
+		error := model.Error{}
```

Comment:
空の構造体作るときは var のがきれいかもしません

```suggestion
		var error model.Error
```

Reply:
```suggestion
		error := model.Error{Message: "Email は必須です。"}
```

でもいいとおもいます！

Reply:
他の箇所に関しても同様です